doc/tutorials/plutus-playground.rst

```diff
@@ -0,0 +1,224 @@
+.. highlight:: haskell
+.. _basic_playground_tutorial:
+
+Compiling and testing a Plutus App in the Plutus Playground
+===========================================================
+
+The Plutus Playground consists of a code editor and a simulator.
+
+.. figure:: images/playground.png
+
+  Plutus Playground (code editor)
+
+We can write Plutus Apps in the code editor and test them in the simulator.
+The Plutus Playground includes a number of sample apps.
+In this tutorial we are going to run through two of the sample apps.
+
+.. _playground-compiling-hello-world:
+
+Running the "Hello, World" app
+------------------------------
+
+When we first open the Plutus Playground, the code editor is populated with the "Vesting" app.
+To load the "Hello, World" app, click the link in the top row.
+This app is much simpler than the vesting app.
+In fact, the app itself only consists of a single line, and a type annotation:
+
+.. literalinclude:: HelloWorldApp.hs
+    :start-after: BLOCK1
+    :end-before: BLOCK2
+
+Compiling the app
+^^^^^^^^^^^^^^^^^
+
+To compile the app, click the "Compile" button.
```

Comment:
It might be worth noting here that the example apps come pre-compiled.

Reply:
Thanks, I forgot about that. I moved the "Compiling the app" section to a note.